runtime/testing-basilisk/Cargo.toml

```diff
@@ -42,6 +42,7 @@ pallet-nft = { path = "../../pallets/nft", default-features = false }
 pallet-lbp = { path = "../../pallets/lbp", default-features = false }
 
 pallet-relaychain-info = { git = "https://github.com/galacticcouncil/warehouse", rev="6b399649f0379bc091ff69447b459ce30dd82b00", default-features = false }
+pallet-faucet = { git = "https://github.com/galacticcouncil/warehouse", rev="11cfa1ae1b77ab4a0b86fcf617b18d9e2511c2e8", default-features = false }
```

Comment:
we cannot use versions just because it is not merged yet?

Reply:
wouldn't it better to use version and branch override in the root?

Reply:
@lumir-mrkva 
Using vesions in this case won't be much helpful. Because you cant really select a version, there will be always only one version available. 

I would go with the precise revision instead of main branch and version.  3 reasons:

1. To avoid potential error when a pallet/crate is updated with breaking changes but it is not yet reflected in basilisk or hydra.
2. Gives you possibility to work on pallets/crate in warehouse repo without the need to immediately update Basilisk or hydra.
3. If you change version of crate in the warehouse repo - but here it is still specified the previous one - the build fails because it wont find it.

Reply:
with proper semver it should work nicely

Reply:
> 
> 
> with proper semver and published crates it should work nicely

yes. with published crates -it would work ok.